Italian/verifica/verifica.md

```diff
@@ -1,6 +1,6 @@
 Mi conto le dita, e la prima volta sono 12,
 la seconda volta sono 7, la terza sono 4,
-mi rendo conto che stao dormendo, ma per
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.